pkg/install/v1alpha1/openebs_crds.go

```diff
@@ -134,6 +134,25 @@ spec:
     # shortNames allow shorter string to match your resource on the CLI
     shortNames:
     - csp
+  additionalPrinterColumns:
+  - JSONPath: .status.capacity.used
+    name: Allocated
+    type: string
+  - JSONPath: .status.capacity.free
+    name: Free
+    type: string
+  - JSONPath: .status.capacity.total
+    name: Size
```

Comment:
Should we have Capacity instead of Size?

Reply:
We can have `Capacity` for pool related properties. Done